addons/mail/static/tests/tours/activity_date_format_tour.js

```diff
@@ -0,0 +1,43 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("mail_activity_date_format", {
+    test: true,
+    steps: () => [
+        {
+            trigger: "button:contains('Activities')",
+        },
+        {
+            trigger: "input[id*='activity_type_id']",
+            run: "click",
+        },
+        {
+            trigger: ".dropdown-item:contains('To-Do')",
+        },
+        {
+            trigger: "div[name='summary'] input",
+            run: "text Go Party",
+        },
+        {
+            trigger: "button:contains('Schedule')",
+        },
+        {
+            trigger: ".o-mail-Activity:contains('Go Party')",
+        },
+        {
+            trigger: ".fa-info-circle",
```

Comment:
Was already there in original PR, but would be nice to have a more specific and meaningful selector.

`fa` icons are not necessarily explicit enough if the name has no business semantics (e.g. `fa-phone` is fine, `fa-circle` is quite broad). And also we could change its icon and/or add more places on screen with similar icon, thus the test is weak to unrelated changes.

Reply:
I changed the selector to `.o-mail-Activity-info i`, rebased the branch  and ran the formatter on the modified files.
It should be ok now :)